tests/parser/types/test_function_returns.py

```diff
@@ -0,0 +1,13 @@
+def test_function_return_list_in_tuple(get_contract_with_gas_estimation):
+    list_tester_code = """
+@public
+def foo() -> (uint256, uint256[2]):
+    return (0, [0, 1])
+    """
+
+    c = get_contract_with_gas_estimation(list_tester_code)
+    assert isinstance(c.foo()[1], list)
+    assert len(c.foo()[1]) == 2
+    assert c.foo()[0] == 0
+    assert c.foo()[1].sort() == [0, 1].sort()
```

Comment:
a single assert of == [0, [0, 1]] is more than enough ;)

Reply:
Actually, that reveals that inner lists returned are reversed. :open_mouth:  Any ideas of what might cause that?

Reply:
I think this is probably a misalignment issue. Try with clearer values like `[5, 6]`.